qemu/tests/sgx_multi_vms.py

```diff
@@ -0,0 +1,46 @@
+from virttest import error_context
+from virttest import env_process
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx multi vms boot test:
+    1. Check host sgx capability
+    2. Boot multi sgx VMs
+    3. Verify each VM sgx enabled in guest
+    4. Check each VM sgx qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Boot four guests", test.log.info)
+    timeout = float(params.get_numeric("login_timeout", 240))
+
+    sgx_cap = SGXHostCapability(test, params)
+    sgx_cap.validate_sgx_cap()
+    host_total_epc_size = sgx_cap.host_epc_size
+    if params.get("monitor_expect_nodes"):
+        sgx_cap.validate_numa_node_count()
+
+    vms = params.get("vms").split()
+    for vm_name in vms:
+        env_process.preprocess_vm(test, params, env, vm_name)
+        vm = env.get_vm(vm_name)
+        vm.verify_alive()
```

Comment:
hi @zixi-chen , I am curious about when the vm is started, as we set start_vm = no, VT will not start vm till we call vm.create(), would you please make a double check?

Reply:
Thanks Zhenchao, I missing change start_vm=yes.